fix(profiling): update install instructions (#6522)
getsentry/sentry-docs: src/platforms/node/common/profiling/index.mdx

```

## Enabling Profiling

> In order to enable profiling, import @sentry/profiling-node, add ProfilingIntegration to your integrations and set profilesSampleRate.

Reviewer: ```suggestion
To enable profiling, import @sentry/profiling-node, add ProfilingIntegration to your integrations, and set the profilesSampleRate.
```
